docs/share-models.md

```diff
@@ -10,9 +10,9 @@ manager: jillfra
 ---
 # How to: Share team completions in Visual Studio IntelliCode
 
-IntelliCode has team completions sharing features that let you share any model you've created and automatically update users that have access when you retrain the model.
+IntelliCode has team completions sharing features that let you share repository-associated team models you've created and automatically update users that have access when you retrain the model.
 
-By creating a team completions model, you get IntelliCode completion recommendations for code that's not in the base model. Sharing a team completions model with collaborators extends the enhanced IntelliSense recommendations to everyone; they don't have to create or retrain a completions model themselves.
+By creating a team completions model, you get IntelliCode completion recommendations for code that's not in the base model. Sharing a team completions model with collaborators extends the enhanced IntelliSense recommendations to everyone; they don't have to create or retrain a team completions model themselves.
 
 ## Share a repository-associated team completions model
 
```

Comment:
the below section should be updated to reflect `Acquire team models for completion` being on by default. Maybe flip it to mention how to opt out

Reply:
Fixed, thanks Viv